packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -180,3 +218,31 @@ export interface ScriptPubKey {
   addresses: string[]
   tokenId: string
 }
+
+export interface MempoolTx {
+  [key: string]: {
+    vsize: number
+    size: number // (DEPRECATED) same as vsize. Only returned if defid is started with -deprecatedrpc=size
```

Comment:
You should do this instead, for certain IDE it will warn quite loudly

```ts
export interface MempoolTx {
  /**
   * @deprecated same as vsize. Only returned if defid is started with -deprecatedrpc=size
   */
  size: number
}